README.md

```diff
@@ -110,7 +110,7 @@ The above can then be executed via the `devlab run` command from within the same
 
 ### Multiple Tasks
 
-The example shows a single-command execution configuration, however, Devlab supports named tasks as well. Replace the `task` entry with configuration object `tasks`:
+The example shows a single-command execution configuration, however, Binci supports named tasks as well. Replace the `task` entry with configuration object `tasks`:

```

Comment:
I may have just completely read that wrong 😁  Anyways, please review to make sure it's what you want and is clear.